src/services/transcription.ts

```
@@ -0,0 +1,163 @@
+    return configPath;
+  }
+
+  // Bare command name: probe via execution to use OS PATH resolution
+  try {
+    await execFileAsync(configPath, ['--help'], { timeout: 5000 });
+    return configPath;
+  } catch (err: unknown) {
+    const e = err as { code?: string; message?: string };
+    // Only fail if executable is truly missing or not executable
+    if (e.code === 'ENOENT' || e.code === 'EACCES') {
+      throw new Error(`Could not resolve ${executableName} in PATH or as executable`);
```

ESLint

[error] There is no `cause` attached to the symptom error being thrown.

(preserve-caught-error)